code/modules/vr/vr_observer.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/device/observer
+	name = "observer"
+	icon = 'icons/obj/items.dmi'
+	desc = "the all seeing eye always watching."
+	icon_state = "observingeye"
+	item_state = "observingeye"
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_EARS
+	flags = NODROP
+	var/on = 0
+	var/obj/machinery/camera/camera
+	var/canhear_range = 7
+
+/obj/item/device/observer/equipped()
+	..()
+	var/mob/living/carbon/human/H = loc
+	if(camera)
+		camera.network = list("news")
+		camera.c_tag = H.name
+	else
+		camera = new /obj/machinery/camera(src)
+		camera.network = list("news")
+		cameranet.removeCamera(camera)
+		camera.c_tag = H.name
+
+/obj/item/device/observer/examine(mob/user)
+	if(..(user, 1))
+		to_chat(user, "This video camera can send live feeds to the entertainment network. It's [camera ? "" : "in"]active.")
```

Comment:
`Its` not `It's`

Also you can use `null` instead of `""` in the ternary operation